index.js

```diff
@@ -124,14 +410,40 @@ instance.prototype.init = function() {
 	log = self.log;
 	self.ptSpeed = '0C';
 	self.ptSpeedIndex = 12;
+	self.commandQueue = [];
+	self.instanceState = {};
+	self.currentCommand = null;
+	self.queuePrune = setInterval( self.pruneCmdQueue.bind(self), 1000 );
```

Comment:
This needs to move to a tcp on connect call and needs to be properly cleared when the socket is not connected for any reason.

Reply:
Ah, very good point. I'll make that change as well.